Task1_UserAPI/MeetingManager.Tests/UserCrudTest.cs

```diff
@@ -0,0 +1,160 @@
+﻿using AutoMapper;
+using MeetingManager.Core.Entities;
+using MeetingManager.Core.Interfaces;
+using MeetingManager.Core.Models;
+using MeetingManager.Core.Services;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MeetingManager.Tests
+{
+    [TestClass]
+    public class UserCrudTest
+    {
+        private IMapper mapper;
+
+        private List<User> users;
+
+        private IUserRepository mockUserRepository;
+
+        private IUserService userService;
+
+        public UserCrudTest()
+        {
```

Comment:
you can use TestInitialize:
  [TestInitialize]
  public void TestInitialize()
{
...
}